.github/scripts/install-slices/install_slices.py

```diff
@@ -315,6 +315,7 @@ def install_slices(
                     release,
                     "--root",
                     tmpfs,
+                    '--ignore=unstable',
```

Comment:
this will be added to installations that use Chisel versions where the flag is not supported